bundle/manifests/osc-alerts_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: osc-alerts
+spec:
```

Comment:
I'm curious about the relation between this file and `config/metrics/metrics-prometheus-rules.yaml`.
Asking because there are minor differences, e.g. this is missing here:

```
  namespace: openshift-sandboxed-containers-operator
```

Reply:
@gkurz @beraldoleal ping on this one. I don't understand how `make bundle` works, and I wonder if the `[osc-alerts_monitoring.coreos.com_v1_prometheusrule.yaml](https://github.com/openshift/sandboxed-containers-operator/pull/476/files/7d0fed1f48a3ac3491d52c8192a02849a30f3a5c..d58528938a126c0225d6b56ed5e6ea8327041202#diff-94b1960d38bbbde8d6a407abb7a64aa6956501d87d40e2cc9c0c2c1950a4181b)` file is generated or if it's manually edited.

Reply:
Hi @c3d, afaict, by design,  k8s resource definitions in the bundle often omit the namespace field because the bundle is meant to be namespace-agnostic. This allows the operator to work in any namespace where it's deployed. The bundle/manifests was generated with just a `make bundle`.